experimental/rocm/tracing.c

```diff
@@ -128,6 +157,23 @@ iree_status_t iree_hal_rocm_tracing_context_allocate(
         timestamp_period);
   }
 
+  context->smutrace_enabled = false;
+  void (*dl) = dlopen(SMUTRACE_DLL, RTLD_LAZY);
```

Comment:
use iree_dynamic_library_load_from_files + iree_dynamic_library_lookup_symbol (search to find examples)